src/System.Private.CoreLib/shared/System/AppDomain.cs

```diff
@@ -184,26 +184,21 @@ public static void Unload(AppDomain domain)
 
         public static bool MonitoringIsEnabled
         {
-            get { return false; }
+            get { return true; }
             set
             {
                 if (!value)
                 {
                     throw new ArgumentException(SR.Arg_MustBeTrue);
                 }
-                throw new PlatformNotSupportedException(SR.PlatformNotSupported_AppDomain_ResMon);
             }
         }
 
-        public long MonitoringSurvivedMemorySize { get { throw CreateResMonNotAvailException(); } }
+        public long MonitoringSurvivedMemorySize => MonitoringSurvivedProcessMemorySize;
 
-        public static long MonitoringSurvivedProcessMemorySize { get { throw CreateResMonNotAvailException(); } }
+        public static long MonitoringSurvivedProcessMemorySize => GC.GetGCMemoryInfo().HeapSizeBytes;
```

Comment:
>GC.GetGCMemoryInfo().HeapSizeBytes [](start = 66, length = 34)

HeapSize also includes Fragmentation so this should be (HeapSize - Fragmentation) which would be survived.

Reply:
Ok, will change, thanks.